PowerEditor/src/Notepad_plus.cpp

```diff
@@ -2567,9 +2567,12 @@ void Notepad_plus::checkClipboard()
 {
 	bool hasSelection = _pEditView->hasSelection();
 	bool canPaste = (_pEditView->execute(SCI_CANPASTE) != 0);
-	//enableCommand(IDM_EDIT_CUT, hasSelection, MENU | TOOLBAR);
-	//enableCommand(IDM_EDIT_COPY, hasSelection, MENU | TOOLBAR);
 
+	if (!NppParameters::getInstance().getSVP()._lineCopyCutWithoutSelection)

```

Comment:
I think you need to get rid of this check.

Based on my observation that copy/cut in the NavigateTo and JsonTools windows __only works when text is selected__ after this commit, the problem would seem to be that when you disable the menu item here, there's never an opportunity for the WM_CUT message to get sent to plugin textboxes.

Reply:
Please check my comment:
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/14660#issuecomment-1924391237